uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -236,6 +237,13 @@ async def asgi_send(self, message):
                 )
                 self.initial_response = None
                 self.accepted_subprotocol = message.get("subprotocol")
+                if "headers" in message:
+                    self.extra_headers.extend(
+                        # ASGI spec requires bytes
+                        # But for compability we need to convert it to strings
+                        (name.decode("latin-1"), value.decode("latin-1"))
+                        for name, value in message.get("headers")
```

Comment:
just small syntax comment:
for name, value in message["headers"]
due to already checking if "headers" in message

Reply:
true, PR welcome :P

Reply:
@t1waz are you going to open PR for that? I can do that also.